sample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uuid" : "path/to/example.tif",
```

Comment:
They could be...my point is that if the URI and the UUID are the same, and the URI is derived from the place that you are reading the metadata, then I don't know why we'd store it in the metadata. If we move the metadata and image pair, we would actually have to modify the json at that point to point to the new location we put the imagery, instead of the URI/UUID of the image file being derived from the read action and so no metadata file change is involved.